### PR TITLE
Making datasets dir env vars consistent

### PR DESCRIPTION
**Pull request by hiteshk-gc**
Making all "dataset_dir" env vars consistent with the naming throughout Graphcore examples, CI and software infra.

Corresponds with:
https://github.com/gradient-ai/Graphcore-Pytorch/pull/53
https://github.com/gradient-ai/Graphcore-Tensorflow2/pull/36

### CODE REVIEW

**Review by stuarts-graphcore — APPROVED** [review]
LGTM
